stdlib/public/core/CTypes.swift

```diff
@@ -227,6 +227,29 @@ extension UInt {
   }
 }
 
+#if arch(arm64) && !(os(macOS) || os(iOS) || os(tvOS) || os(watchOS))
```

Comment:
This should also include Windows, as Windows ARM64 handles variadics similar to macOS.